### ISSUE TITLE
Enhancement: Add Zerodha Console functionality

### ISSUE THREAD

**Issue by KanchangC** [jugaad-py/jugaad-trader#15]
can you please add Zerodha console access too?

**Comment by sevakram** [comment]
Yes, I was thinking of creating a new Console class. But there is no fixed timeline for it

**Comment by sevakram** [comment]
Update: It has been included in current git branch. But documentation is pending.

**Comment by KanchangC** [comment]
thanks for prompt response. I can see that you have added one API key for console. is that zerodha kite API key? (paid subscription?)

**Comment by sevakram** [comment]
@iamknownstranger , While the PR #23 solved the Websocket issue, but it broke the Zerodha Console feature I was building 😄 . 

Now the tests for the Console class are failing.

**Comment by sevakram** [comment]
> thanks for prompt response. I can see that you have added one API key for console. is that zerodha kite API key? (paid subscription?)

No, Each Zerodha app ie. Console, Coin etc will have an API key. Probably they use the same APIs for their products as well.
